### PR TITLE
updated copy

### PR DESCRIPTION
**Pull request by lauraxm**

### CODE REVIEW

**Comment by lauraxm** [comment]
https://trello.com/c/RFBPQnGS/126-tweaks-from-lateesha

**Comment by ghost** [comment]
💃

File: index.html [hunk 1]
```diff
@@ -527,7 +526,7 @@ <h2 class="section-title">LOCATION</h2>
 				<h2 class="section-title">Sponsors</h2>
 			</div>
 			<div class="col-8 col">
-				<p class="qcTabDesc">Sponsorship is the most effective way to recruit talented, driven women to fill key technical positions and demonstrate your commitment to the professional development of women at your organization. <a href="assets/sponsor-prospectus.pdf" target="_blank" class="link coral">Read our 2017 Prospectus</a></p>
+				<p class="qcTabDesc">Sponsorship is the most effective way to recruit talented, driven women to fill key technical positions and demonstrate your commitment to the professional development of women at your organization.<br <a href="assets/sponsor-prospectus.pdf" target="_blank" class="link coral">Read our 2017 Prospectus</a></p>
```
Comment (jarmstrng) [inline index.html]: Ooh, looks like we're missing a `>` on the break - the link doesn't work

**Review by jarmstrng — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (jarmstrng) [reply index.html]: I think that comment pinged you but just in case: @laumontemayor

**Comment by lauraxm** [comment]
@jarmstrng checked and looks good on staging on Safari | Chrome | IE | Firefox | Brave 💃

**Review by jarmstrng — APPROVED** [review]

**Comment by jarmstrng** [comment]
@laumontemayor Looks good! Please merge into master & deploy master to prod, let me know if you have any issues (though I'll be on a boat tonight, so if prod breaks just roll it back 😂 )

**Pull request merged**